src/leds.rs

```diff
@@ -305,6 +306,11 @@ impl LedManager {
             let new_host_status = Self::current_host_status();
 
             if new_vtarget != current_vtarget {
+                // A change in Vtarget means that a cable was re- or disconnected, in
+                // which case knowing the core status is impossible, so we just reset
+                // it to the default.
+                Self::set_host_status(None);
```

Comment:
How precise is vtarget? does this not accidentally trigger?

Reply:
In my experience it is plenty precise for this: the probe/board combo I'm working with measure around 3350 mV, +- 20 mV. Would be good to get some more data on it, that is true

Reply:
The update loop is [here](https://github.com/probe-rs/rusty-probe-firmware/blob/master/src/bin/app.rs#L72).

AFAIK there are bigger problems than the LED if it's not measure correctly, as it would also mean that the voltage translators would jump around a bunch

Reply:
That's a good question. I have no idea how Vtarget works in practice, and how/if there are any specifications that say "it really shouldn't do voltage drops". Let's ask @korken89 :D

Reply:
Hmm, I'd recommend to check within a span from nominal instead of != :)

Reply:
My fault this was the set value, so it checks out

Reply:
We check for the voltage once every 100 ms, and only act if it drops below 2500 mV or 1700 mV before propagating a change in VTarget.